src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Area[] figure = new Area[6];
+        for (int i = 0; i < figure.length; i++) {
+            if (i < figure.length / 2) {
+                figureSupplier.getRandomFigure();
+            }
+            if (i > figure.length / 2) {
```

Comment:
1. if (i == figure.length / 2) -> do nothing? Please use if/else for more consistency instead of 2 if()
2. Please add figures to created array
3. Please move here logic with calling draw method